src/Controller/CustomerController.php

```diff
@@ -32,21 +36,30 @@ public function __construct(CustomerRepository $customerRepository)
 
      public function addCustomer(Request $request): JsonResponse
      {
-         $data = json_decode($request->getContent(), true);
+        
+            $data = json_decode($request->getContent(), true);
+        
+            try {
+                $firstName = $data["firstName"];
+                $lastName = $data["lastName"];
+                $email = $data["email"];
+                $phoneNumber = $data["phoneNumber"];
+            } catch (ErrorException $e) {
+                return new JsonResponse(["message" => "Enter value for all field"], Response::HTTP_BAD_REQUEST);
+            }
+
+            if (empty($firstName) || empty($lastName) || empty($email) || empty($phoneNumber)){
+                throw new NotFoundHttpException("Expecting mandatory parameters:");
```

Comment:
Check formatting in this function body. Seems like more spaces than needed

Reply:
yeah, removed in next commit